dts/msm8916/msm8916-huawei-hx4qcom.dts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <206 0>,<248 0>,<249 0>,<250 0>;
```

Comment:
https://github.com/torvalds/linux/blob/efb448d0a3fca01bb987dd70963da6185b81751e/drivers/soc/qcom/socinfo.c#L110-L113

Keep only `<206 0>`(MSM8916/LTE) if there is no 3G/Wi-Fi only variant.